sample_keyprovider/tools/generate_keys.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e
+declare -A keys
+
+usage() {
+    cat << EOT
+This script aims to generate a "aa-offline_fs_kbc-keys.json" for offline-fs-kbc,
+or export a specific key in side "aa-offline_fs_kbc-keys.json" to binary for
+image encryption.
+	Usage:
+		- ./generate_keys.sh generate
+			Generate a "aa-offline_fs_kbc-keys.json" with randomly generated KEKs inside.
+		- ./generate_keys.sh export <path-to-aa-offline_fs_kbc-keys.json> <key-uri> <output-path>
+			Export the key of <key-uri> in the given "aa-offline_fs_kbc-keys.json". The result
+			key will be exported to <output-path>
+EOT
+    exit
+}
+
+create_keys() {
+	for i in $(seq 1 "$1"); do
+		keys[test-repo/key/key_id$i]=$(head -c32 < /dev/random | base64)
```

Comment:
Maybe there should be an option to set the repo that will be used.

Reply:
Yes, added